operator/cmd/root.go

```diff
@@ -148,10 +148,11 @@ func initEnv() {
 	option.LogRegisteredOptions(Vp, log)
 	// Enable fallback to direct API probing to check for support of Leases in
 	// case Discovery API fails.
-	option.Config.EnableK8sLeasesFallbackDiscovery()
+	//option.Config.EnableK8sLeasesFallbackDiscovery()
```

Comment:
did you mean to comment this out?

Reply:
This along with few other changes in this patch were fixed in a later patch. I didn't want that one patch to include large changes in both `daemon/cmd` and `operator/cmd`, so instead just commented out the parts that would not compile.

Reply:
aha, gotcha.